docs/changelog.md

```diff
@@ -12,6 +12,10 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+* **browser-domains**
+  * New domain autocomplete providers `ShippedDomainsProvider` and `CustomDomainsProvider` that
+    should be used instead of deprecated `DomainAutoCompleteProvider`. 
```

Comment:
nit: This needs to be moved to the latest version now. :)

Reply:
It is in `0.54.0-SNAPSHOT`. :)